cognite/neat/graph/extractors/_classic_cdf/_assets.py

```diff
@@ -74,6 +119,8 @@ def _asset2triples(self, asset: Asset, namespace: Namespace) -> list[Triple]:
         type_ = "Asset"
         if self.to_type:
             type_ = self.to_type(asset) or type_
+        type_ = self._SPACE_PATTERN.sub("_", type_)
```

Comment:
see my comment about potential ill-formed URIs due to illegal characters beyond space.
Option could be to form URI safe , but ideally remove/replace any none websafe/dms-safe character with e.g. "_"